src/Nethermind/Nethermind.Synchronization/FastBlocks/ReceiptsSyncFeed.cs

```diff
@@ -170,7 +170,8 @@ private bool TryPrepareReceipts(BlockInfo blockInfo, TxReceipt[] receipts, out T
                 }
                 else
                 {
-                    IReleaseSpec releaseSpec = _specProvider.GetSpec(blockInfo.BlockNumber);
+                    // BlockInfo has no timestamp
+                    IReleaseSpec releaseSpec = _specProvider.GetSpec((ForkActivation)blockInfo.BlockNumber);
```

Comment:
okay, but the proposed solution is really bad. We will pass incorrect releaseSpec. Please, resolve it better